### ISSUE TITLE
Thanks for this utility!

### ISSUE THREAD

**Issue by mlongval** [keaston/cp437#4]
Works great for connecting to old BBSs via telnet on the Linux CLI. 

Thank you very much. 

Cheers from Canada!

**Comment by cmang** [comment]
I agree. I was scratching my head thinking about how something like this could be written, and I found that @keaston has done the work. This is a great utility.

I wonder if any BBSs are using it to translate CP437 door games like LORD to Utf8, for playing in modern terminals.
